code/game/objects/items/melee/misc.dm

```diff
@@ -230,6 +230,26 @@
 /obj/item/melee/classic_baton/Initialize()
 	. = ..()
 
+// For when we want to swap from Brute damage to Stamina and vise versa
+/obj/item/melee/classic_baton/AltClick(mob/user)
+	. = ..()
+	if(damtype == "brute")
+		damtype = STAMINA
+		user.visible_message("<span class='notice'>[user] wields the [src] is a non-lethal way.</span>")
+		return
+	if(damtype == STAMINA)
+		damtype = "brute"
+		user.visible_message("<span class='warning'>[user] wields the [src] is a lethal way!</span>")
```

Comment:
"Trilby wields the Classic Baton **is** a lethal way!"

Awh heck I think you accidentally typed "is" instead of "in".

Reply:
Thanks <3